variables.tf

```diff
@@ -13,7 +13,7 @@ variable "create_namespace" {
 variable "chart_version" {
   type        = string
   description = "Chart version to be deployed."
-  default     = "1.0.12"
+  default     = "1.0.13"
```

Comment:
@yehielnetapp 
Don't forget to change the readme file
"Chart version to be deployed..."

Reply:
In addition, in the HELM chart, you've allowed the client to modify the strategy parameters (how??)
And here we can't perform it - let's talk about it